HRIS.Models/EmployeeQualificationDto.cs

```diff
@@ -0,0 +1,23 @@
+﻿using HRIS.Models.Enums.QualificationEnums;
+using System.ComponentModel.DataAnnotations;
+
+namespace HRIS.Models;
+
+public class EmployeeQualificationDto
+{
+    public int Id { get; set; }
+    public int EmployeeId { get; set; }
+
+    [Required]
+    public HighestQualification HighestQualification { get; set; }
+    [Required]
+    public string? School { get; set; }
```

Comment:
Does this ? not make the attribute nullable?

Reply:
which line?

Reply:
All the fields for the qualification are required.

Reply:
I see the error in my code. I will make an update to make all the fields non-nullable values.